project-fortis-interfaces/src/components/Insights/MapBoundingReset.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import IconButton from 'material-ui/IconButton/IconButton';
+import Map from 'material-ui/svg-icons/maps/map';
+import { fullWhite } from 'material-ui/styles/colors';
+
+export default class MapBoundingReset extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      expanded: false
+    };
+  }
+
+  onClick = () => {
+    this.props.onClick();

```

Comment:
If you're only going to delegate to `this.props.onClick` you might as well just reference that method in the IconButton below. `this.props.onClick` will be bound to the correct this-context just like `this.onClick` class-method is here. If you'd like to learn more about the this-binding, I really liked [this article](https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56) on the subject.